Tests/DependencyInjection/Compiler/AnnotationCompilerPassTest.php

```diff
@@ -55,6 +55,8 @@ public function testEmbeddedAdmin()
             $meta->tags['sonata.admin'][0],
             array(
                 'manager_type'      => 'orm',
+                'group'             => 'Admin',
+                'label'             => 'Tests\Fixtures\Foo',
```

Comment:
this should be ok (based on the regex change of https://github.com/sonata-project/SonataAdminBundle/pull/3687#discussion_r57702074)

this is just a fallback for the admin name